src/provisioner/config_store.py

```diff
@@ -44,3 +51,25 @@ def get(self, key: str) -> str:
         """ Returns value for the given key """
 
         return Conf.get(self._conf_idx, key)
+
+    def _get_kvs(self, prefix, conf_value):
```

Comment:
Fundamental is to assume that everything is key value in string format. Assuming non string is the problem.

Reply:
Removed this code, as it is not required anymore.